.github/workflows/tests.yml

```diff
@@ -94,3 +94,13 @@ jobs:
         run: |
           make --always-make test-data
           git diff --exit-code -- ./test/data
+
+  codespell:
+    name: "Spell check"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codespell-project/actions-codespell@master
+        with:
+          ignore_words_list: msdos
+          skip: ./.git,coverity,rpmbuild,samples
```

Comment:
"rpmbuild" is not a thing anymore ;)

Reply:
Ah yeah, not in CI, but locally. I am torn between keeping them sync or having it CI specific.

Reply:
I don't mind either. I usually prefer when the CI just invokes `make foo`, because then we can easily execute it locally as well. And then it makes sense to me to keep both in sync. But in this case, the github-action cannot really be executed locally, can it? So I don't mind dropping it. But really not much of an issue either way.